ocaml/xapi/jbuild

```diff
@@ -112,4 +113,11 @@ let () = Printf.ksprintf Jbuild_plugin.V1.send {|
   (action (run ${<} -runner sequential -verbose true))
  )
 )
+
+(alias
+ ((name runalcotest)
```

Comment:
I think you can make both of these run as part of runtest if you put alcotest in another folder

Reply:
How can I do that?

Reply:
See https://discuss.ocaml.org/t/jbuilder-support-for-multiple-test-executables/825 you can in fact have two `runtest` in same jbuild file.

Reply:
Thanks, that's great, I'll use two!